service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/SampleRoboticsTestDataHelper.java

```diff
@@ -24,7 +25,7 @@ private SampleRoboticsTestDataHelper() {
     public static RoboticsData expectedRoboticsData(final String applicationType) {
         return RoboticsData.builder()
             .caseNumber("12345")
-            .applicationType(applicationType)
+            .applicationType(WordUtils.capitalize(applicationType, new char[] {' '}))
```

Comment:
do not format it here, as this is expected data test should pass expected value

Reply:
this will be deleted as we will now be using the string values instead of enum values